LoopCore/NSUserDefaults.swift

```diff
@@ -20,7 +20,7 @@ extension UserDefaults {
         case lastProfileExpirationAlertDate = "com.loopkit.Loop.lastProfileExpirationAlertDate"
         case allowDebugFeatures = "com.loopkit.Loop.allowDebugFeatures"
         case allowSimulators = "com.loopkit.Loop.allowSimulators"
-        case studyProductSelection = "com.loopkit.Loop.studyProductSelection"
+        case studyProductSelection = "org.tidepool.plugins.TidepoolSupport.StudyProductSelection"
```

Comment:
This doesn't need to be here, anymore, right?

Reply:
It's used within the `LoopAppManager` to keep the selected study product after resetting Loop.

```swift
private func resetLoopUserDefaults() {
  // Store values to persist
  let allowDebugFeatures = UserDefaults.appGroup?.allowDebugFeatures
  let studyProductSelection = UserDefaults.appGroup?.studyProductSelection // HERE
  
  // Wipe away whole domain
  UserDefaults.appGroup?.removePersistentDomain(forName: Bundle.main.appGroupSuiteName)
  
  // Restore values to persist
  UserDefaults.appGroup?.allowDebugFeatures = allowDebugFeatures ?? false
  UserDefaults.appGroup?.studyProductSelection = studyProductSelection  // HERE
}
```

Reply:
Okay this logic was moved to `TidepoolSupport`